package/opt/hassbian/suites/homeassistant.sh

```diff
@@ -161,6 +161,16 @@ fi
 echo "Restarting Home Assistant"
 systemctl restart home-assistant@homeassistant.service
 
+echo "Waiting for Home Assistant to start"
+for i in 1 2 3 4 5
```

Comment:
This is a tad cleaner since it's a more common for a common loop.
```
for i in {1..5}
    do
    sleep 1s
    if [ $(systemctl is-active home-assistant@homeassistant.service) == "active" ]; then
       break
    fi
done
```

Reply:
Thanks - old C programmers first shell simple shell script. Will update.